playground/infrastructure/cd_helper.py

```diff
@@ -0,0 +1,13 @@
+from ci_helper import CIHelper
```

Comment:
add magic comment, licence and module doc

Reply:
Changed.